client-app/assets/styles/main.scss

```diff
@@ -2,59 +2,20 @@
 @tailwind components;
 @tailwind utilities;
 
-@import "helpers";
-@import "print";
-@import "vendors";
-
 @layer base {
-  body {
-    @apply text-[--color-body-text];
+  html, :host {
+    @apply font-lato;
   }
```

Comment:
I found that there is difference in classes on body in `liquid`, used for production (environments) and `index.html`, used for local development (and storefront-less configuration in future). `font-lato` is missed on first:
https://github.com/VirtoCommerce/vc-theme-b2b-vue/blob/ec59c90530c6dbf5340478c70d371b3bd5c8fe4d/layout/theme.liquid#L9
https://github.com/VirtoCommerce/vc-theme-b2b-vue/blob/ec59c90530c6dbf5340478c70d371b3bd5c8fe4d/index.html#L16
I wanted to make PR and bring it into `index.html`, but than found you did this change and now I propose to remove it from `liquid` in this PR instead.